docs/topics/auth/passwords.txt

```diff
@@ -693,6 +693,10 @@ or if you have API calls that allow passwords to be set, for example.
 Note that where the password is passed to one of these functions, this should
 always be the clear text password - not a hashed password.
 
+If you would like to override the default error messages if a password fails
+to validate, you can subclass the built-in validator classes and override the
+``get_error_message()`` method.
```

Comment:
Thanks for adding the documentation. One thing I noted is that the ``get_error_message()`` method sometimes takes an extra argument (most take no extra arguments, but ``UserAttributeSimilarityValidator.get_error_message()`` requires ``verbose_name``). Is there a good way to document this, so that people who override the method don't end up with an unexpected error?
Maybe we can give an example for overriding each of the 4 classes?

Reply:
Thanks - I added some basic examples for overriding the classes.

Reply:
Thanks for adding those! The documentation looks good to me, though you're welcome to change the example if you think using ``verbose_name`` in the example is a good idea